qargparse.py

```diff
@@ -145,24 +139,36 @@ def _addArgument(self, arg):
             widget.setAttribute(QtCore.Qt.WA_StyledBackground)
             widget.setEnabled(arg["enabled"])
 
-        # Align label on top of row if widget is over two times heiger
+        # Reset btn widget
+        _reset = QtWidgets.QWidget()
+        _reset.setProperty("type", "QArgparse:reset")
+        reset = QtWidgets.QPushButton("")  # default
+        reset.setToolTip("Reset")
+        reset.hide()  # shown on edit
+
+        # Align label on top of row if widget is over two times higher
         height = (lambda w: w.sizeHint().height())
         label_on_top = height(label) * 2 < height(widget)
         alignment = (QtCore.Qt.AlignTop,) if label_on_top else ()
 
+        # Layout
+        layout = QtWidgets.QVBoxLayout(_reset)
```

Comment:
This layout doesn't seem necessary, could we not just add `reset` directly to the layout below, like before? What does `_reset` add that cannot be done with `reset`?

Reply:
> And yes, if we want the argument widget don't get stretched while reset button is being hidden or shown, then we do need the reset_container to take the property "QArgparse:reset" and being like a fixed frame for reset button.

Ah, is this it? We couldn't add the property straight to `reset.setProperty(..)`?

Reply:
Oh yes, the property has changed the assignment to the `_reset` (or `reset_container`) QWidget object . And here's a gif to show the different 👍🏼 

Before :
![f2a9fb41-7dcd-4c4e-90c1-fa2fee122dc6](https://user-images.githubusercontent.com/3357009/89046702-f00e5580-d37f-11ea-9322-10949b28d8e1.gif)

After :
![1ffca869-1a0e-4de8-8d3a-7da55fa63701](https://user-images.githubusercontent.com/3357009/89047070-7165e800-d380-11ea-9f49-48c382585aa1.gif)

Reply:
Ah, excellent. Then I'm happy, thanks David.